modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceCurrencies.testcase

```diff
@@ -14,6 +14,100 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-11393. Can update currency format for Indian Rupee"
+	@priority = 3
+	test CanUpdateCurrencyFormatForINR {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site with a user without a role is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			JSONUser.addUser(
+				userEmailAddress = "testuser@liferay.com",
+				userFirstName = "test",
+				userLastName = "user",
+				userScreenName = "testuser");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "testuser@liferay.com");
+
+			JSONUser.addUserToSite(
+				groupName = "Minium",
+				userEmailAddress = "testuser@liferay.com");
+		}
+
+		task ("When the format pattern for Indian Rupee currency is updated to '₹ ##,##,##0.00'") {
+			CommerceNavigator.goToCurrenciesAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Indian Rupee");
+
+			Type(
+				key_fieldLabel = "Format Pattern",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "₹ ##,##,##0.00");
+
+			Button.clickSave();
+		}
+
+		task ("And a simple product with base price '1000000' is created") {
+			CommerceJSONProductsAPI._addCommerceProduct(
+				catalogName = "Minium",
+				productName = "Simple Product",
+				productType = "Simple");
+
+			CommerceProducts.openProductsAdmin();
+
+			CommerceProducts.searchProduct(search = "Simple");
+
+			CommerceNavigator.gotoEntry(entryName = "Simple Product");
+
+			CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+			CommerceNavigator.gotoEntry(entryName = "Simple Product_default");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+			Type(
+				key_fieldLabel = "Base Price",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = 1000000);
+
+			Button.clickPublish();
```

Comment:
This can be done using JSON macros, please see `CommerceJSONProductsAPI._addCommerceProduct();`